vms/platformvm/state/state.go

```diff
@@ -834,9 +893,14 @@ func (s *state) GetSubnetOwner(subnetID ids.ID) (fx.Owner, error) {
 		return nil, fmt.Errorf("%q %w", subnetID, errIsNotSubnet)
 	}
 
+	s.SetSubnetOwner(subnetID, subnet.Owner)
```

Comment:
This feels a bit weird (to be performing a write on the read)... But I think it's ok